critiquebrainz/ws/review/test/views_test.py

```diff
@@ -186,3 +190,33 @@ def test_revision_entity_handler(self):
         self.client.post('/review/%s' % review["id"], headers=self.header(self.user), data=json.dumps(data))
         resp = self.client.get('/review/%s/revisions/2' % review["id"])
         self.assert200(resp)
+
+    def test_cache_tracking(self):
+        entity_id = self.review["entity_id"]
+        track_key = cache.gen_key("ws_cache", entity_id)
+
+        # Test no cache if entity id is not provided
+        self.client.get('/review/', query_string={'sort': 'rating'})
+        cache_keys = cache.get(track_key, namespace="Review")
+        self.assertEqual(None, cache_keys)
```

Comment:
What is this checking? I think it makes sense that there's no cache item that includes entity_id (from a few lines earlier). Does it create any cache items in this case? If it creates no cache items then I think we should instead mock cache set and ensure that it's not called. Otherwise I don't think that it makes too much sense to have this test here

Reply:
This part was for testing that if a query does not have an entity_id, it does not cached. We are now reconsidering what to do in this case.